server/controllers/lecture_booking-controller.js

```diff
@@ -0,0 +1,52 @@
+const knex = require("../db");
+const moment = require("moment");
+
+//Insert new booking of a lecture
+
+exports.newBooking = async (req, res) => {
+    // Insert new booking from table lecture_booking
+    const studentId = req.user && req.user.id;
+    console.log(req.user);
+    today = moment().format('YYYY-MM-DD HH:mm:ss');
+
+    knex("lecture_booking")
+      .insert({
+        // insert new record
+        lecture_id: req.body.lecture_id,
+        student_id: studentId,//idstudent  
+        booked_at: today  //time
+      })
+      .then(() => {
+        // Send a success message in response
+        res.json({ message: `Booking created.` });
+      })
+      .catch((err) => {
+        // Send a error message in response
+        res.json({ message: `There was an error creating the booking` });
+      });
+  };
+
+//Get existent bookings by one student
+
+exports.getExistentBooking = async (req, res) => {
```

Comment:
Same as above, probably this method is more suited for the `user-controller`.

Reply:
Yeah, maybe this is more suited for the user-controller

Reply:
Put it on lecture just to be consistent with other methods (getBookingLectures, newBooking)